aws-api/src/androidTest/java/com/amplifyframework/api/aws/RestApiInstrumentationTest.java

```diff
@@ -118,4 +146,29 @@ public void getRequestWithApiKey() throws JSONException {
                 "/simplesuccessapikey",
                 contextJSON.getString("resource-path"));
     }
+
+    /**
+     * Test whether we can make api Rest call in IAM as auth type.
+     */
+    @Test
+    public void getRequestWithIAM() {
+        final RestOptions options = RestOptions.builder().addPath("items").build();
+        LatchedRestResponseListener responseListener = new LatchedRestResponseListener();
+        Amplify.API.get("iamAPI", options, responseListener);
+        RestResponse response = responseListener.awaitTerminalEvent().awaitSuccessResponse();
+        assertTrue("Should return a non null data", response.getData() != null);
```

Comment:
I think this might already be inside of `awaitSuccessResponse`, can you check?

If not, this is more simply worded in jUnit as `assertNotNull(response.getData())`;.